sql/indexes.sql

```diff
@@ -105,6 +105,7 @@ create index IX_4F40FE5E on DLFileEntryMetadata (fileEntryId);
 create unique index IX_BE290777 on DLFileEntryMetadata (fileVersionId, ctCollectionId, DDMStructureId);
 create index IX_D49AB5D1 on DLFileEntryMetadata (uuid_[$COLUMN_LENGTH:75$]);
 
+create index IX_D2F8189A on DLFileEntryType (companyId);
```

Comment:
hey @jkappler ,
Can this somehow include ctCollectionId so that Publications can also leverage this improvement?